website/docs/r/cdn_frontdoor_origin_group.html.markdown

```diff
@@ -0,0 +1,119 @@
+---
+subcategory: "CDN"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_cdn_frontdoor_origin_group"
+description: |-
+  Manages a Frontdoor Origin Group.
+---
+
+# azurerm_cdn_frontdoor_origin_group
+
+Manages a Frontdoor Origin Group.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-cdn-frontdoor"
+  location = "West Europe"
+}
+
+resource "azurerm_cdn_frontdoor_profile" "example" {
+  name                = "example-profile"
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_cdn_frontdoor_origin_group" "example" {
+  name                         = "example-originGroup"
+  cdn_cdn_frontdoor_profile_id = azurerm_cdn_frontdoor_profile.example.id
+
+  health_probe {
+    interval_in_seconds = 240
+    path                = "/healthProbe"
+    protocol            = "Https"
+    request_type        = "GET"
+  }
+
+  load_balancing {
+    additional_latency_in_milliseconds = 0
+    sample_size                        = 16
+    successful_samples_required        = 3
+  }
+
+  session_affinity                      = true
+  restore_traffic_or_new_endpoints_time = 10
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Frontdoor Origin Group. Changing this forces a new Frontdoor Origin Group to be created.
+
+* `cdn_frontdoor_profile_id` - (Required) The ID of the Frontdoor Profile. Changing this forces a new Frontdoor Origin Group to be created.
+
+* `load_balancing` - (Required) A `load_balancing` block as defined below.
+
+* `health_probe` - (Optional) A `health_probe` block as defined below.
+
+* `session_affinity` - (Optional) Whether to allow session affinity on this host. Possible values are `true` or `false`. Defaults to `true`.
+
+* `restore_traffic_or_new_endpoints_time` - (Optional) Time in minutes to shift the traffic to another endpoint when an healthy endpoint becomes unhealthy or a new endpoint is added. Default is `10` minutes.
```

Comment:
we should include the unit in the property name
```suggestion
* `restore_traffic_or_new_endpoints_after_min` - (Optional) Time in minutes to shift the traffic to another endpoint when an healthy endpoint becomes unhealthy or a new endpoint is added. Default is `10` minutes.
```

Reply:
Fixed